.travis.yml

```diff
@@ -3,17 +3,17 @@ sudo: false
 matrix:
     include:
         - os: linux
-          env: DEPS="numpy==1.11.0 nose" PYTHON_VERSION="2.7" PYTHON_ARCH="64"
+          env: DEPS="numpy==1.11.0 nose cython" PYTHON_VERSION="2.7" PYTHON_ARCH="64"
```

Comment:
Since you're pinning the numpy version it might be a good idea to do that for Cython as well.